doc/ref_model/chapters/appendix-a.md

```diff
@@ -16,19 +16,75 @@ This Appendix has two goals:
+
+**1. Infrastructure**:
+- a. Application functionality or application control require infrastructure components beyond RM profiles or requires infrastructure configuration changes beyond RA exposed APIs. Generally, such application is tightly coupled with infrastructure which results in Appliance deployment model. 
+- b. Application control using RA APIs finds node with required infrastructure component(s), and in that node using RA APIs configures infrastructure components that make application work. Example is application that requires certain acceleration adapter available in RM profile and is exposed through RA APIs.
+- c. Application control using RA APIs finds node with optional infrastructure component(s), and in that node using RA APIs configures infrastructure component(s) that make application work better (like more performant) than without that infrastructure component. Example is app that would work better with certain acceleration adapter but can also work without it.
+- d. Application control using RA APIs finds general profile node without any specific infrastructure component.
+
+**2. Platform as a Service** (as per RA2 WIP: common services starting from Load Balancer and Store State, also typically includes Logging and Telemetry):
+- a. Application functionality or application control can work only with its own components implementing platform services already provided by CNTT-defined PaaS.
+- b. With custom integration effort, application can be made to use CNTT-defined PaaS.
+- c. Application is designed and can be configured for running with CNTT-defined PaaS.
+
+**3. Application Resiliency** (Note: “resilient” is the 1st word describing “enabled loosely coupled systems” in CNCF Cloud Native Definition [6])
+- a. Application was designed and tested to run only on Carrier Grade platform with predictable infrastructure availability and performance
+- b. Application was designed and tested for HW and SW component full failures but not for infrastructure impairment which still needs predictable infrastructure performance (like CPU cycles and network latencies)
+- c. Application was designed to run on shared Cloud platforms and tested for resilience to infrastructure impairments
+
+Relevant for sizing infrastructure and application operations (which often is another telco organizational unit or external 3rd party running on telco cloud) is also how much is application decomposed from:
+
+**4. Other application functionality** (decomposition, manageability):
+- a. Application consists of huge monolithic components including algorithms that have different scaling (for example depending on type of traffic) and/or availability requirements
+- b. Application is decomposed into smaller components that are tightly coupled by same versioning (upgrading all components or none)
```

Comment:
consistent with (a) above and for simplification, can we change to "Application consists of smaller tightly coupled components that all need to be upgraded together." Version numbers do not necessarily need to be the same but all components need to be upgraded.

Reply:
Done in #1376 .